src/components/Dropdown/Dropdown.tsx

```diff
@@ -90,12 +90,13 @@ export class Dropdown extends React.PureComponent<IDropdownProps, IDropdownState
         const dropdownIconClassName = hasTitleBorder ? 'iconArrowWithBorder' : 'iconArrow';
 
         const selectionTextObj = this.getSelectionText(dropdownType, selectedOption);
-
+        const isError = this.props.validated === undefined ? false : !this.props.validated;
```

Comment:
you can put validated: true in defaultProps to avoid this check. ie. it could be used somewhere else in the future so we do not have to repeat this logic.

Reply:
Added default prop and changed the prop name for better understanding